brownie/_cli/accounts.py

```diff
@@ -1,88 +1,75 @@
-
 import json
 import shutil
-import sys
 from pathlib import Path
 
+import click
+
 from brownie import accounts
 from brownie._config import _get_data_folder
 from brownie.convert import to_address
 from brownie.utils import color, notify
-from brownie.utils.docopt import docopt
-
-__doc__ = """Usage: brownie accounts <command> [<arguments> ...] [options]
-
-Commands:
-  list                             List available accounts
-  new <id>                         Add a new account by entering a private key
-  generate <id>                    Add a new account with a random private key
-  import <id> <path>               Import a new account via a keystore file
-  export <id> <path>               Export an existing account keystore file
-  password <id>                    Change the password of an existing account
-  delete <id>                      Delete an existing account
-
-Options:
-  --help -h                        Display this message
-
-Command-line helper for managing local accounts. You can unlock local accounts from
-scripts or the console using the Accounts.load method.
-"""
 
 
-def main():
-    args = docopt(__doc__)
-    try:
-        fn = getattr(sys.modules[__name__], f"_{args['<command>']}")
-    except AttributeError:
-        print("Invalid command. Try brownie accounts --help")
-        return
-    try:
-        fn(*args["<arguments>"])
-    except TypeError:
-        print(f"Invalid arguments for command '{args['<command>']}'. Try brownie accounts --help")
-        return
+@click.group(short_help="Manage local accounts")
+def cli():
+    """
+    Command-line helper for managing local accounts. You can unlock local accounts from
+    scripts or the console using the Accounts.load() method.
+    """
 
 
+# Different name because `list` is a keyword
+@cli.command(name="list", short_help="List available accounts")
 def _list():
     account_paths = sorted(_get_data_folder().glob("accounts/*.json"))
-    print(f"Found {len(account_paths)} account{'s' if len(account_paths)!=1 else ''}:")
-    for path in account_paths:
-        u = "\u2514" if path == account_paths[-1] else "\u251c"
-        with path.open() as fp:
-            data = json.load(fp)
-        print(
-            f" {color('bright black')}{u}\u2500{color('bright blue')}{path.stem}{color}"
-            f": {color('bright magenta')}{to_address(data['address'])}{color}"
-        )
-
-
-def _new(id_):
+    if len(account_paths) == 0:
+        print("No accounts found.")
+    else:
+        print(f"Found {len(account_paths)} account{'s' if len(account_paths)!=1 else ''}:")
+        for path in account_paths:
+            u = "\u2514" if path == account_paths[-1] else "\u251c"
+            with path.open() as fp:
+                data = json.load(fp)
+            print(
+                f" {color('bright black')}{u}\u2500{color('bright blue')}{path.stem}{color}"
+                f": {color('bright magenta')}{to_address(data['address'])}{color}"
+            )
```

Comment:
`click.echo` should be preferred over `print`

Reply:
I have discussed this with ben, and I think we either switch to echo **everywhere** or leave in print. I am open to either solution. Using echo in some places and print in other most likely does more harm than good.

Reply:
Using click will make it easier to support Windows. I know that's not necessarily something Ben cares about at this point, but perhaps at some point in the future it will be useful. Also could refactor the color system to use ASCII color codes. It's not as featureful, but has broader support across platforms.